snuba/clickhouse/query.py

```diff
@@ -1,8 +1,13 @@
 from typing import NewType
 
+from snuba.query.expressions import Expression as SnubaExpression
 from snuba.query.logical import Query as LogicalQuery
 
 # This defines the type for the Clickhouse query that operates on storages.
 # Obviously this is supposed to become more complex than just a copy of the
 # logical query type.
 Query = NewType("Query", LogicalQuery)
+
+# This defines the type of a Clickhouse query expression. It is used to make
+# the interface of the expression translator more clear.
+Expression = NewType("Expression", SnubaExpression)
```

Comment:
This really exists only to make this easier to understand so that we know exactly what depends on the logical expression and what depends on the clickhouse expression.